### PR TITLE
Feat/article list video label

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Added video icon to article summary package

screen shot 2018-06-15 at 13 51 05

### CODE REVIEW

File: packages/video-label/src/video-label.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import beautifyTitle from "./beautify-title";
 
 const VideoLabel = ({ title, color }) => (
   <View style={{ flexDirection: "row", alignItems: "center" }}>
-    <IconVideo height={styles.title.fontSize} fillColor={color} />
+    <View style={styles.iconContainer}>
+      <IconVideo height={styles.icon.iconSize} fillColour={color} />
```
Comment (craigbilner) [inline packages/video-label/src/video-label.js]: should height === iconSize?

Comment (tuncaulubilge) [reply packages/video-label/src/video-label.js]: Was equal to fontSize, but it ends up being bigger than the text itself, probably due to sans serif font.

File: packages/article-summary/src/styles/shared.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const sharedStyles = {
     marginBottom: spacing(1)
   },
   labelWrapper: {
-    marginBottom: 0
+    marginBottom: spacing(1)
```
[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply packages/article-summary/src/styles/shared.js]: Yes, it's intentional and review by Jake. We needed some spacing between the headline and the label

File: packages/article-summary/fixtures/video-label.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    title: "Camilla Long",
+    color: colours.functional.primary,
+    isVideo: true
+  },
+  datePublicationProps: {
+    date: "2017-11-17T00:01:00.000Z",
+    publication: "TIMES"
+  },
+  headline: () => (
+    <ArticleSummaryHeadline headline="Top medal for forces dog who took a bite out of the Taliban" />
```
Comment (colincclark) [inline packages/article-summary/fixtures/video-label.js]: 🐶

File: packages/article-summary/src/article-summary.js [hunk 4]
```diff
@@ -75,7 +87,8 @@ const ArticleSummary = props => {
 ArticleSummary.propTypes = {
   labelProps: PropTypes.shape({
     title: PropTypes.string.isRequired,
-    color: PropTypes.string
+    color: PropTypes.string,
+    isVideo: PropTypes.bool
```
Comment (colincclark) [inline packages/article-summary/src/article-summary.js]: these should have defaults

File: packages/video-label/src/style/index.web.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import sharedStyles from "./shared";
+
+const styles = {
+  title: {
```
Comment (colincclark) [inline packages/video-label/src/style/index.web.js]: better to destruct `... sharedStyles` and override that rather than destruct each and every style

Comment (tuncaulubilge) [reply packages/video-label/src/style/index.web.js]: 73d467e

File: packages/article-summary/src/article-summary.js (hunk 4 above)
Comment (tuncaulubilge) [reply packages/article-summary/src/article-summary.js]: 73d467e

**Pull request merged**